### ISSUE TITLE
HotkeySas does'nt work!

### ISSUE THREAD

**Issue by frankieorabona** [petersohn/SASHotkeys#1]
I tried HotkeySas BUT it doesn't work...

If I install it and then launch KSP in the initial screen nothing works...I click but nothing happens.

Where do I go wrong?

**Comment by petersohn** [comment]
It might not be compatible with the newest KSP version. I haven't been looking at it for a long time. Sorry for that.

**Comment by frankieorabona** [comment]
First of all, thanks for your reply...
Oh that' s a shame!  I needed it to assign keys to a USB controller...are you not planning to update it?

**Comment by petersohn** [comment]
No, right now I'm not playing the game. But if I ever decide to play again, I will update it.
